Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -1625,6 +1625,303 @@ function Test-DiscoveryService{
     }
 }
 
+function Migrate-AADSettings {
+    param (
+        [Parameter(Mandatory=$true)]
+        [string] $installConfigPath,
+        [Parameter(Mandatory=$true)]
+        [string] $azureConfigPath
+    )
+    $configSection = "identity"
+    $nodesToSearch = @("tenants", "replyUrls", "claimsIssuerTenant", "allowedTenants", "registeredApplications")
+    $childNodeGetAttribute = "name"
+    $childNodeSetAttribute = "value"
+
+    Write-DosMessage -Level "Information" -Message "Get the AAD Setting contents in install.config"
```

Comment:
It is more to know where the migration process got to, if we had to troubleshoot. Maybe I will change the wording to be AAD Setting check complete, so the user doesn't think there is an issue.

Reply:
The other option is to move this to verbose.  In general, take an logs that are too chatty for the typical end user and move them to debug or verbose since they still have lots of value for troubleshooting.

Reply:
Did that, not seeing them logged after setting minimumLoggingLevel = "Verbose"